metaworld/envs/mujoco/env_dict.py

```diff
@@ -496,92 +487,155 @@ def _hard_mode_args_kwargs(env_cls_, key_):
+          ('faucet-close-v2', SawyerFaucetCloseEnvV2),
+          ('hammer-v2', SawyerHammerEnvV2),
+          ('handle-press-side-v2', SawyerHandlePressSideEnvV2),
+          ('handle-press-v2', SawyerHandlePressEnvV2),
+          ('handle-pull-side-v2', SawyerHandlePullSideEnvV2),
+          ('handle-pull-v2', SawyerHandlePullEnvV2),
+          ('lever-pull-v2', SawyerLeverPullEnvV2),
+          ('peg-insert-side-v2', SawyerPegInsertionSideEnvV2),
+          ('pick-place-wall-v2', SawyerPickPlaceWallEnvV2),
+          ('pick-out-of-hole-v2', SawyerPickOutOfHoleEnvV2),
+          ('reach-v2', SawyerReachEnvV2),
+          ('push-back-v2', SawyerPushBackEnvV2),
+          ('push-v2', SawyerPushEnvV2),
+          ('pick-place-v2', SawyerPickPlaceEnvV2),
+          ('plate-slide-v2', SawyerPlateSlideEnvV2),
+          ('plate-slide-side-v2', SawyerPlateSlideSideEnvV2),
+          ('plate-slide-back-v2', SawyerPlateSlideBackEnvV2),
+          ('plate-slide-back-side-v2', SawyerPlateSlideBackSideEnvV2),
+          ('peg-insert-side-v2', SawyerPegInsertionSideEnvV2),
+          ('peg-unplug-side-v2', SawyerPegUnplugSideEnvV2),
+          ('soccer-v2', SawyerSoccerEnvV2),
+          ('stick-push-v2', SawyerStickPushEnvV2),
+          ('stick-pull-v2', SawyerStickPullEnvV2),
+          ('push-wall-v2', SawyerPushWallEnvV2),
+          ('push-v2', SawyerPushEnvV2),
+          ('reach-wall-v2', SawyerReachWallEnvV2),
+          ('reach-v2', SawyerReachEnvV2),
+          ('shelf-place-v2', SawyerShelfPlaceEnvV2),
+          ('sweep-into-v2', SawyerSweepIntoGoalEnvV2),
+          ('sweep-v2', SawyerSweepEnvV2),
+          ('window-open-v2', SawyerWindowOpenEnvV2),
+          ('window-close-v2', SawyerWindowCloseEnvV2),
+      ))), ('test',
+            OrderedDict((
+                ('bin-picking-v2', SawyerBinPickingEnvV2),
+                ('box-close-v2', SawyerBoxCloseEnvV2),
+                ('hand-insert-v2', SawyerHandInsertEnvV2),
+                ('door-lock-v2', SawyerDoorLockEnvV2),
+                ('door-unlock-v2', SawyerDoorUnlockEnvV2),
+            )))))
 
 ml45_train_args_kwargs = {
-    key: dict(args=[], kwargs={
-        'task_id' : list(ALL_V2_ENVIRONMENTS.keys()).index(key),
-    })
+    key: dict(args=[],
+              kwargs={
+                  'task_id': list(ALL_V2_ENVIRONMENTS.keys()).index(key),
+              })
     for key, _ in ML45_V2['train'].items()
 }
 
 ml45_test_args_kwargs = {
-    key: dict(args=[], kwargs={'task_id' : list(ALL_V2_ENVIRONMENTS.keys()).index(key)})
+    key: dict(args=[],
+              kwargs={'task_id': list(ALL_V2_ENVIRONMENTS.keys()).index(key)})
     for key, _ in ML45_V2['test'].items()
 }
 
 ML45_ARGS_KWARGS = dict(
     train=ml45_train_args_kwargs,
     test=ml45_test_args_kwargs,
 )
+
+
+def create_hidden_goal_envs():
+    hidden_goal_envs = {}
+    for env_name, env_cls in ALL_V2_ENVIRONMENTS.items():
+        d = {}
+
+        def initialize(cls, seed=None):
```

Comment:
Can you rename `cls` to `env` or `self`?

Reply:
sure. The function that we pass this function to expects the first parameter to be cls though @krzentner

Reply:
fixed @krzentner